docs/source/noise/index.rst

```diff
@@ -38,45 +36,55 @@ Currently, errors added in different columns are independent of each other.
 Available Noise Types
 ---------------------
 
-Pseudopeople applies the different types of row-based and column-based noise in
-the following order to mimic the data generation process by which a real dataset
-might be corrupted. The table lists all the available noise types, but not every
+The table lists all the available noise types, but not every
 type of noise will necessarily be applied to every dataset or every column. The
 configuration determines which noise types are actually used. The "Noise Type"
 column shows the name of the noise type in the configuration system.
 
-.. list-table:: Types of noise in order of application
-  :widths: 1 2 5 1
+.. list-table:: Types of row-based noise (``row_noise``)
+  :widths: 1 2 5 
   :header-rows: 1
 
   * - Noise Type
-    - Description
-    - Example
-    - Order
-  * - ``row_noise``
-    - **Row-based noise**
-    - ---
-    - ---
-  * - ``omit_row``
-    - Omit a row
+    - Config key 
+    - Example cause
+  * - Omit a row
+    - ``omit_row``
     - Neglecting to file a tax form on time
-    - 1
-  * - ``column_noise``
-    - **Column-based noise**
-    - ---
-    - ---
-  * - ``leave_blank``
-    - Leave a field blank
+
+.. list-table:: Types of column-based noise (``column_noise``)
+  :widths: 1 2 5 
+  :header-rows: 1
+
+  * - Noise Type
+    - Config key
+    - Example cause
+  * - Leave a field blank
+    - ``leave_blank``
     - Forgetting to write your name on the designated line
-    - 2
-  * - ``choose_wrong_option``
-    - Choose the wrong option from a fixed set of options
+  * - Choose the wrong option
+    - ``choose_wrong_option``
     - Marking the "Male" box when you meant "Female"
-    - 3
-
-.. todo::
+  * - Optical character recognition (OCR) error
+    - ``make_ocr_errors``
+    - Misreading an 'S' instead of a '5'
+  * - Make typos
```

Comment:
If we're sticking with listing things in the order of application, typos should be last. Here's the order we specified in the [concept model](https://vivarium-research.readthedocs.io/en/latest/models/concept_models/vivarium_census_synthdata/concept_model.html#noise-functions), with the ones we've implemented in **bold** (and using the old names, not the new names): 

1. **“Borrowed” SSN (happens in simulation NOT noise functions)**
3. **Omissions**
5. Duplications
7. **Missing Data**
9. **Incorrect Selection**
11. Copy from Within Household
13. Month and day swaps (applies to dates only)
15. **Zip Code Miswriting (applies to Zip Code only)**
17. **Age Miswriting (applies to age only)**
19. **Numeric miswriting**
21. Nicknames
23. **Fake names**
25. Phonetic
27. OCR
29. **Typographic**

We talked about re-arranging things a bit, e.g.,

1. **“Borrowed” SSN (happens in simulation NOT noise functions)**
3. **Omissions**
5. Duplications
7. **Missing Data**
9. **Incorrect Selection**
11. Copy from Within Household
21. Nicknames
23. **Fake names**
13. Month and day swaps (applies to dates only)
19. **Numeric miswriting**
15. **Zip Code Miswriting (applies to Zip Code only)**
17. **Age Miswriting (applies to age only)**
25. Phonetic
27. OCR
29. **Typographic**

Or something like that. Should we choose an order and stick with it? Should we document the changes in the concept model (perhaps with our updated naming conventions as well), or is there a better place?

Reply:
I think we should use the re-arranged order you just suggested in this table, and then once we've filled in the details for all noise types, re-arrange the sections into that order.

Also, just to note -- "happens in simulation NOT noise functions" is not going to be meaningful to our users. Instead we should say it is "not configurable."

Reply:
Wait actually -- shouldn't "age miswriting" aka "age misreporting" happen before numeric miswriting? I think they both apply to the age column, so it is a meaningful difference.

Reply:
According to the concept model, we didn't ask for numeric miswriting noise on age:
<img width="593" alt="image" src="https://user-images.githubusercontent.com/16144326/234171406-8df8a792-ae2a-441c-b991-77db3c20f71c.png">
Similarly, zipcodes get zipcode miswriting but not basic numeric miswriting. I think the idea was that these specialized functions would replace numeric miswriting for these particular columns. But I don't necessarily see a reason we should _prevent_ a user from applying both types of noise (age + numeric or zip + numeric) if they want to. In which case I think you're right that it probably makes more sense to do the specialized function first and numeric miswriting after.